.github/workflows/test_package.yml

```diff
@@ -22,12 +22,10 @@ jobs:
       run: "python -m pip install -q --upgrade pip wheel"
     - name: Install dependencies
       run: |
-        brew install zeromq flatbuffers
-        sudo wget https://raw.githubusercontent.com/zeromq/cppzmq/v4.3.0/zmq.hpp -P \
-          /usr/local/include
+        brew install cmake
     - name: Clone repo
       uses: actions/checkout@v2
-    - name: Install from repo in test mode
+    - name: Install nle via pip
       run: "pip install nle"
     - name: Check nethack is installed
-      run: "type -p nethack"
+      run: "python -c 'import nle; import gym; gym.make(\"NetHack-v0\")'"
```

Comment:
Is there any test associated with this or do we simply expect it not to crash?

Reply:
This is the test for the package and we just hope it doesn't crash but instead is able to load the environment.